### PR TITLE
wireplumber: update to 0.4.5

### PR DESCRIPTION
**Pull request by st3r4g**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

**Comment by st3r4g** [comment]
this version runs also without `libspa-v4l2` so I'm removing the `depends=`

**Comment by ericonr** [comment]
Feel free to adopt it, if you want to track it together with main pipewire. I am currently using it, so keeping maintainer as me is also fine.

File: srcpkgs/wireplumber/template [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Template file for 'wireplumber'
 pkgname=wireplumber
-version=0.4.4
+version=0.4.5
 revision=1
 build_style=meson
 build_helper=gir
 configure_args="-Dintrospection=enabled -Dsystem-lua=true -Dwpipc=enabled"
```
Comment (paper42) [inline srcpkgs/wireplumber/template:7]: according to the changelog of 0.4.2

> - wpipc has been removed and split out to a separate project, https://git.automotivelinux.org/src/pipewire-ic-ipc/

Comment (ericonr) [reply srcpkgs/wireplumber/template:7]: Was it ever part of the package? I remember that changelog entry, don't think there was anything to act on.

Comment (paper42) [reply srcpkgs/wireplumber/template:7]: We should at least remove this option, meson 0.60 will make unknown options like this fatal errors

Comment (ericonr) [reply srcpkgs/wireplumber/template:7]: I forgot to actually scroll, sorry. Hadn't seen the option.

**Pull request merged**